manifests/7/7zip/7zip/19.00.00.0/7zip.7zip.installer.yaml

```
@@ -52,31 +52,31 @@ Installers:
   - Architecture: x64
     InstallerLocale: en-US
     InstallerType: msi
-    InstallerUrl: https://raw.githubusercontent.com/ip7z/a/main/7z1900-x64.msi
+    InstallerUrl: https://www.7-zip.org/a/7z1900-x64.msi
     InstallerSha256: A7803233EEDB6A4B59B3024CCF9292A6FFFB94507DC998AA67C5B745D197A5DC
     ProductCode: "{23170F69-40C1-2702-1900-000001000000}"
     Scope: machine
     UpgradeBehavior: install
   - Architecture: x64
     InstallerLocale: en-US
     InstallerType: nullsoft
-    InstallerUrl: https://raw.githubusercontent.com/ip7z/a/main/7z1900-x64.exe
-    InstallerSha256: 0f5d4dbbe5e55b7aa31b91e5925ed901fdf46a367491d81381846f05ad54c45e
+    InstallerUrl: https://www.7-zip.org/a/7z1900-x64.exe
+    InstallerSha256: 0F5D4DBBE5E55B7AA31B91E5925ED901FDF46A367491D81381846F05AD54C45E
     Scope: machine
     UpgradeBehavior: install
   - Architecture: x86
     InstallerLocale: en-US
     InstallerType: msi
-    InstallerUrl: https://raw.githubusercontent.com/ip7z/a/main/7z1900.msi
+    InstallerUrl: https://www.7-zip.org/a/7z1900.msi
     InstallerSha256: B49D55A52BC0EAB14947C8982C413D9BE141C337DA1368A24AA0484CBB5E89CD
     ProductCode: "{23170F69-40C1-2701-1900-000001000000}"
     Scope: machine
     UpgradeBehavior: install
   - Architecture: x86
     InstallerLocale: en-US
     InstallerType: nullsoft
-    InstallerUrl: https://raw.githubusercontent.com/ip7z/a/main/7z1900.exe
-    InstallerSha256: 759aa04d5b03ebeee13ba01df554e8c962ca339c74f56627c8bed6984bb7ef80
+    InstallerUrl: https://www.7-zip.org/a/7z1900-x64.msi
+    InstallerSha256: A7803233EEDB6A4B59B3024CCF9292A6FFFB94507DC998AA67C5B745D197A5DC
```

```suggestion
    InstallerUrl: https://www.7-zip.org/a/7z1900.exe
    InstallerSha256: 759AA04D5B03EBEEE13BA01DF554E8C962CA339C74F56627C8BED6984BB7EF80
```
x64 MSI already exists at https://github.com/microsoft/winget-pkgs/pull/27816/files#diff-bfcd069bd661db240b98cdf0d421263cca6e455d74d85975a3d390707cd70ce3R55